packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -142,29 +142,31 @@ var convertDate = function (dateString) {
 var alreadyValidDateTime = function (dateTimeString) {
   if (!dateTimeString || dateTimeString.toString() === "") return false;
   var ds = dateTimeString.toString();
-  return /^(\d{4})-(\d{2})-(\d{2})(?:[ T](\d{2}):(\d{2}):(\d{2})(?:\.(\d{1,7}))?(Z|[-+]\d{2}:?\d{2})?)?$/.test(
+  return /^(\d{4})-(\d{2})-(\d{2})(?:[T](\d{2}):(\d{2}):(\d{2})(?:\.(\d{1,7}))?(Z|[-+]\d{2}:?\d{2})?)?$/.test(
     ds
   );
 };
 
 const incompeteIso8601Regex = /^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}$/;
-const Iso8601RegexWithMissingT = /^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}(\.\d{3}Z)$/;
+const iso8601RegexWithMissingT = /^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}(\.\d{3}Z)$/;
 
 var incompleteIso8601 = function (dateTimeString) {
-  if (incompeteIso8601Regex.test(dateTimeString) || Iso8601RegexWithMissingT.test(dateTimeString)) {
+  if (!dateTimeString || dateTimeString.toString() === "") return false;
```

Comment:
nit: from...
 
```ts
... dateTimeString.toString() === ""
```

...to

```ts
... dateTimeString.trim().length() < 1
```

It might actually be worth to have it as a function so we're not rewriting this over and over again.

Reply:
Added this